docs/releases/1.11.1.txt

```diff
@@ -9,4 +9,6 @@ Django 1.11.1 fixes several bugs in 1.11.
 Bugfixes
 ========
 
-* ...
+* Made migrations respect ``Index``\'s ``name`` argument. If you created a
```

Comment:
I don't think `\'` renders the quote in the correct direction, I used a right quote in my suggestion.